packages/client/mutations/AddAgendaItemMutation.ts

```diff
@@ -21,11 +21,12 @@ graphql`
       }
     }
     meeting {
+      ...useGotoStageId_meeting
       phases {
         ...ActionSidebarAgendaItemsSectionAgendaItemPhase @relay(mask: false)
         stages {
           ...ActionMeetingAgendaItemsStage @relay(mask: false)
-          # For useMeetingGotoStageId
+          # For useGotoStageId
           isNavigableByFacilitator
```

Comment:
-1 This is part of the `useGotoStageId_meeting` fragment and can be removed here.

Reply:
this was here before my changes so removing that could break _something_ unless we add `@relay(mask: false)`.

Reply:
Fair point, then please update the comment accordingly.

Reply:
as `useMeetingGotoStageId` doesn't exist I guess it's best to just remove it